src/hotspot/share/gc/shenandoah/shenandoahHeap.hpp

```diff
@@ -480,6 +484,12 @@ class ShenandoahHeap : public CollectedHeap {
   inline size_t set_young_evac_reserve(size_t new_val);
   inline size_t get_young_evac_reserve() const;
 
+  // Methods related to age tables in generational mode for Young Gen
+  inline ShenandoahAgeCensus* age_census() const;
+  AgeTable* get_age_table();
+  void update_epoch();
+  void reset_epoch();
```

Comment:
`epoch` overload/confusion with prior use. Consider if this is really needed; if so, find better names to avoid confusion with other uses of the term `epoch` or methods thereof.

Reply:
Done.